### DISCUSSION TITLE
图标

### DISCUSSION THREAD

**Discussion by berebrn** [constanline/XQuickEnergy#252] [Show and tell/凡尔赛]
植物🌵🌲🌳🌴🌿🍀☘️🌱🍃🎋🌾🎍🌺🌸🌼🏵️🌻🌹🌷💐🥀🍂🍁

食物🍅🍆🍉🍊🍎🍓🍔🍘🍙🍚🍛🍜🍝🍞🍟🍡🍢🍣🍦🍧🍰🍱🍲🍳🍵☕🍶🍺🍻🍸🍏🍐🍋🍌🍇🍈🍒🍑🍍🌽🌶️🍠🌰🍯🧀🍤🍗🍖🍕🌭🌮🌯🍥🍨🍮🍭🍬🍫🍿🍩🍪🍼🍷🍹🍾🥛🥞🥖🥐🎂🥙🥗🥕🥒🥔🥝🍄🥑🥜🥓🥚🥘🥃🚰💊🥂

手势💋👀👁️👅👄👣🤝🤛🤜🤞🤙🤳👌👍👎💅🙏🤚🖐️️✌️️👊✍️️✊👆🖕👇👈🤘👉✋👃🙌👏👋🖖☝️️👐👂💪

心形❤️💛💚💙💜🖤💔❣️💕💞💓💗💖💘💝

节日🎉🎊✨🎈🎁🎃🎄🛶🐲⛄☃️🤶🎅

交通🚗🚕🚙🚌🚎🏎️🚓⛴️🚑🚒🚐🚚🚛🚜🛴🚲🛵🏍️🚔🚍🚘🚖🚡🚠🚟🚃🚋🚞🚝🚄🚅🚈🚂🚆🚇🚊🚉✈️🛫🛬🛩️🛰️🚀🚁⛵🚤🛥️🛳️🚢🚦🚥🚧⛽⚠️🚸🚏

运动🚵‍♂️🚵‍♀️🚴‍♂️🚴‍♀️🏄‍♂️🏄‍♀️🏊‍♂️🏊‍♀️🤽‍♂️🤽‍♀️⛹️‍♀️⛹️‍♂️🤸‍♀️🤸‍♂️🏇🤾‍♂️🤾‍♀️🤼‍♂️🤼‍♀️🤺🏂⛷️🏌️‍♀️🏌️‍♂️🚶‍♂️🚶‍♀️🏃‍♂️🏃‍♀️🏋️‍♂️🏋️‍♀️🚣‍♂️🚣‍♀️🕺💃🏆🥇🥈🥉🏹🎣🏅⛸️🏓🎿🏑🏐⚽🎾🏈🏉🏀🥊🎱🎳⚾🏏🏒🏸🥅⛳

物品⌚📱📲💻⌨️🖥️🖨️🖱️🖲️🕹️🎮🗜️💽📀💿💾📼📷📸🎬📹📽️🎥📞🎞️☎️📈📉📊📟📠📺📻🎙️🎚️🎛️⏱️⚙️⏰⏲️🕰️⌛⏳📡💡🔦🔌💄🛢️💸💴💶💵💷💰💳💎⚖️🔧🔨⚒️🛠️⛏️🔩🔫💣🔪🗡️⚔️🛡️🚬⚰️⚱️🏺🔮📿💈🔋⚗️🔭🕳️🔬🌡️💉🚽🚿🛁🛀🛎️🔑🗝️🚪🛋️🛏️🛌🛍️🛒🎎🏮🎐✉️📩📧💌📨📤📥📦🏷️📭📬📪📫📮📯📜📃📄📑🗒️📆🗓️📅🗑️📇🗃️🗳️🗄️📋📁📂🗂️🗞️📰📓📔📒📕📗📘📙📚📖🔖🔗📎🖇️📐📏📌📍✂️🖊️🖋️✒️🖌️🖍️📝✏️🔍🔎🔏🔐🔒🔓🎟️🎫🗺️🎠🚨🎯🔰♟️🃏🎴🎰🎲🀄🔈📣📢🎶🎤🥁🎺🎸🎷🎻🎹🎧〽️🔱🐚💮🌂☂️⛱️⛺🎭💺⛓️⚓💠🎨🗿🍽️🍴🥄🎖️

风景🗽🗼🏟️🎪🎡🎢🏜️⛲🏖️🏝️🏕️🛤️🛣️🗻⛰️🏔️🌋🌌🌁🌃🗾🌅🌆🎇🎆🌉🌄🌠🌇🎑🏙️🏞️🖼️🌎🌍🌏

天气🌞🌝🌛🌜🌚🌕🌖🌗🌘🌔🌓🌒🌑🌙☀️🌤️⛅🌥️☁️🌦️🌧️⛈️🌩️🌨️❄️🌟⚡💧☔🌈🌊🌫️🌪️☄️

职业👮‍♀️👮‍♂️👷🏽‍♀️👷🏾‍♂️💂‍♀️💂‍♂️🕵️‍♀️🕵️‍♂️👩‍⚕️👨‍⚕️👩‍🌾👨‍🌾👩‍🍳👨‍🍳👩‍🎓👨‍🎓👩‍🎤👨‍🎤👩‍🏫👨‍🏫👩‍🏭👨‍🏭👩‍💻👨‍💻👩‍💼👨‍💼👩‍🔧👨‍🔧👩‍🔬👨‍🔬👩‍🎨👨‍🎨👩‍🚒👨‍🚒👩‍✈️👨‍✈️👩‍🚀👨‍🚀👩‍⚖️👨‍⚖️👰🤵♻️🔥💨🗯️💭💬🌬️

**Comment by berebrn** [comment]
在TXT里能打出来的以上就是

**Comment by berebrn** [comment]
[emoji.txt](https://github.com/constanline/XQuickEnergy/files/12407461/emoji.txt)

**Comment by kimshift** [comment]
没有动物图标么？

**Comment by berebrn** [comment]
有
动物🐶🐱🐭🐹🐰🦊🐻🐼🐨🐯🦁🐮🐷🐽🐸🐵🙊🙉🙈🐒😺😸😹😻😼😽🙀😿😾🐔🐧🐦🐤🐣🐥🦆🦅🦉🦇🐺🐗🐴🦄🐝🐛🦋🐌🐞🐜🕷️🕸️🦂🐢🐉🐍🦎🐙🦑🦐🦀🐡🐠🐟🐬🐳🐋🦈🐊🐅🐆🦍🐘🦏🐪🐫🐂🐃🐄🐎🐖🐏🐑🐐🦌🐕🐩🐈🐓🦃🕊️🐇🐁🐀🐿️🐾
